src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1697,20 +1712,41 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   </Target>
 
   <Target Name="GetTargetFrameworks"
+          DependsOnTargets="GetTargetFrameworksWithPlatformForSingleTargetFramework"
           Returns="@(_ThisProjectBuildMetadata)">
+
+    <Error Condition="'$(IsCrossTargetingBuild)' == 'true'"
+           Text="Internal MSBuild error: Non-CrossTargeting GetTargetFrameworks target should not be used in cross targeting (outer) build" />
```

Comment:
Looking at https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1257284, does that mean this error check shouldn't be here because you can have an inner build that invokes an outer build in a way that it still needs to get the inner build's target frameworks and match them with the new outer build's target frameworks?